src/encoded/audit/file.py

```diff
@@ -335,6 +335,27 @@ def audit_duplicate_quality_metrics(value, system):
                 level='INTERNAL_ACTION'
             )
 
+def audit_file_output_category(value, system):
```

Comment:
Two spaces between functions unless they are in a class.  Same with below the function.